mappings/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -203,6 +203,19 @@ CLASS net/minecraft/server/MinecraftServer
 		ARG 3 force
 			COMMENT when set to true, all the {@link ServerWorld}s will be saved even if {@link ServerWorld#savingDisabled} is set to true
 	METHOD method_3724 isSinglePlayer ()Z
```

Comment:
Should we rename this method to `isSingleplayer`? It's whether singleplayer mode or not.

Reply:
Sure. Just hope this isn't too extraneous.